jcl/src/java.base/share/classes/java/lang/invoke/ArrayVarHandle.java

```diff
@@ -119,9 +119,25 @@
 	}
 /*[ENDIF] Java12 */
 
+/*[IF Java16]*/
+	public MethodType accessModeTypeUncached(AccessType type) {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+	}
+/*[ELSE]*/
 	public MethodType accessModeTypeUncached(AccessMode accessMode) {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
+/*[ENDIF] Java16 */
```

Comment:
The method name isn't changed, just the argument type which can be JPP decorated instead.
```suggestion
	public MethodType accessModeTypeUncached(
/*[IF Java16]*/
           AccessType type
/*[ELSE]*/
	   AccessMode accessMode
/*[ENDIF] Java16 */
	) {
		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
	}
```

Can this be moved into `java.lang.invoke.VarHandle` to avoid duplicate code within `FieldVarHandle` and `ViewVarHandle`?